src/main/java/com/wolox/training/model/User.java

```diff
@@ -0,0 +1,126 @@
+package com.wolox.training.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.sun.istack.NotNull;
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+
+@Entity(name = "Usser")
+public final class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long idUser;
+    @NotNull
+    private String username;
+    @NotNull
+    private String name;
+    @NotNull
+    private LocalDate birthdate;
+    @OneToMany
```

Comment:
use `@ManyToMany`, one user can be associated with many books and one book can be associated with many users